### DISCUSSION TITLE
Suggested Edit Mode

### DISCUSSION THREAD

**Discussion by lancedouglas1** [toeverything/AFFiNE#6149] [Ideas]
Hi again AFFiNE team,

I've been test-driving AFFiNE self-hosted. However, I have noticed a missing feature which I believe would significantly boost the platform's appeal:

**Suggested Edit Mode**: The beauty of a 'Suggested Edit' is the respectful editing it promotes. Users can propose changes to documents, with the original text marked but still visible/available, and everyone can review or acknowledge (and comment on hopefully one day) the updates accordingly. This feature encourages seamless and interactive editing, enriching the overall content review process.

Adding this feature could enrich the user experience of AFFiNE, making it an even more tantalizing collaboration tool. 

Thanks for your time and consideration - I'm excited to see how AFFiNE continues to develop in response to user feedback!

**Comment by EYHN** [comment]
This is what we call the `Comments` feature and it’s already on the way

**Reply by lancedouglas1** [reply]
Thanks @EYHN, this one is more like editing with approvals than commenting.

However, where can I track progress on the `Comments`?  I don't see it other than a non-detailed note on https://github.com/toeverything/blocksuite/issues/3121

I also posted an idea for Commenting here: #6148
